docs/_docs/languages/hack.md

```diff
@@ -111,31 +108,28 @@ For example, here is a bit of code that looks relatively haphazard from a format
 
 ![](/static/images/docs/language-hack-badly-formatted.png)
 
-If your cursor is inside the function and you `cmd-shift-C` (`ctrl-shift-C` on Linux), your will be
-formatted.
+Place your cursor inside the function and press `Cmd-Shift-C` (`Ctrl-Shift-C` on Linux) to apply the coding standards to the function.
 
 ![](/static/images/docs/language-hack-well-formatted.png)
 
 ## Debugging
 
-Nuclide has support for debugging PHP and Hack projects.
-[HHVM](https://docs.hhvm.com/hhvm/installation/introduction) is required for debugging
-Hack and PHP code.
+Nuclide has support for debugging PHP and Hack projects. [HHVM](https://docs.hhvm.com/hhvm/installation/introduction) is required for debugging Hack and PHP code.
```

Comment:
Let's change "required for debugging" to "recommended for debugging".  Theoretically our PHP debugger works for zend as well, since both Zend and HHVM speak Xdebug.